src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final char W_CHARACTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName));) {
+            StringBuilder wordsReader = new StringBuilder();
+            int value = reader.read();
+            if (value == -1) {
+                return new String[0];
+            }
+            int i = 0;
+            int counter = 0;
+            while (value != -1) {
+                if ((value >= 65 && value <= 122) || (char) value == ' ') {
```

Comment:
your solution is quite hardcoded at the moment
let's use `reader.readLine()` to read each line
then we can split this line and gather the filtered words into a list or a string

both list and string could then be converted to array and sorted